### PR TITLE
improved readability and features

### PR DESCRIPTION
**Pull request by kjellski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,19 +40,47 @@
...
+      crossorigin="anonymous"
+    ></script>
+
     <script
       type="text/javascript"
       src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"
     ></script>
+
+    <script type="text/javascript" src="./script.js"></script>
+    <script id="playlists" type="x-tmpl-mustache">
```
Comment (kjellski) [inline index.html:61]: Nice templating, much mustache!

File: playlistItems.json [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+   "id": "UExRN3V1Vlk2TTlpZnhTWDlfdkFxSVBIQWhDWXFpNGR1di41NkI0NEY2RDEwNTU3Q0M2",
+   "snippet": {
+    "publishedAt": "2017-03-23T17:14:49.000Z",
+    "channelId": "UCQepnZmhCT9Boc2jbv7ehug",
+    "title": "Raiffeisen Serviceportal - Login",
+    "description": "",
+    "thumbnails": {
+     "default": {
+      "url": "https://i.ytimg.com/vi/1Ip-D6Ch-YU/default.jpg",
+      "width": 120,
```
Comment (kjellski) [inline playlistItems.json:21]: Wayyyy to big!?!

**Review by kjellski — COMMENTED** [review]
LGTM

**Review by jmoeller93 — APPROVED** [review]
Approved

**Pull request merged**